vision/google/cloud/vision/face.py

```diff
@@ -84,19 +119,38 @@ def __init__(self, joy_likelihood, sorrow_likelihood,
         self._anger_likelihood = anger_likelihood
 
     @classmethod
-    def from_api_repr(cls, response):
-        """Factory: construct `Emotions` from Vision API response.
+    def from_api_repr(cls, emotions):
+        """Factory: construct ``Emotions`` from Vision API response.
+
+        :type emotions: dict
+        :param emotions: Response dictionary representing a face.
+
+        :rtype: :class:`~google.cloud.vision.face.Emotions`
+        :returns: Populated instance of ``Emotions``.
+        """
+        joy_likelihood = Likelihood[emotions['joyLikelihood']]
+        sorrow_likelihood = Likelihood[emotions['sorrowLikelihood']]
+        surprise_likelihood = Likelihood[emotions['surpriseLikelihood']]
+        anger_likelihood = Likelihood[emotions['angerLikelihood']]
```

Comment:
Yup. [Can be seen here](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/4f5f782b0cc64920cf63a185b49a430b463d3199/vision/google/cloud/vision/likelihood.py#L21).